.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## What changes are proposed in this pull request?
+ 
+(Please fill in changes proposed in this fix)
+ 
+## How is this patch tested?
+ 
+(Details)
+ 
+## Release Notes
+ 
+### Is this a user-facing change? If so, please give a 1-2 sentence description for MLflow users.
+ 
+(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
+ 
+### What component(s) does this PR touch?
+ 
+- [ ] UI
+- [ ] CLI 
+- [ ] API 
+- [ ] REST-API 
+- [ ] Examples 
+- [ ] Docs
+- [ ] Tracking
+- [ ] Projects 
+- [ ] Artifacts 
+- [ ] Models 
+- [ ] Scoring 
+- [ ] Serving
+- [ ] R
+- [ ] Java
+- [ ] Python
+
+### Please add a label to the PR so it can be classified correctly in the release notes. Options:
```

Comment:
We could instead put this info in the CONTRIBUTING.md file and refer to it if we think every PR is going to end up with this section in the PR description... :-)

Reply:
I think this seems like a good place to add the label; I think it's difficult to expect users to read a markdown file that isn't automatically displayed in the repo home page.